WordPress/Classes/Utility/Migrator/CoreDataIterativeMigrator.swift

```diff
@@ -4,11 +4,60 @@ import CoreData
 /// CoreDataIterativeMigrator: Migrates through a series of models to allow for users to skip app versions without risk.
 ///
 class CoreDataIterativeMigrator: NSObject {
+    static let versionInfoPlist = "VersionInfo.plist"
+    static let versionHashesKey = "NSManagedObjectModel_VersionHashes"
 
     private static func error(with code: IterativeMigratorErrorCodes, description: String) -> NSError {
         return NSError(domain: "IterativeMigrator", code: code.rawValue, userInfo: [NSLocalizedDescriptionKey: description])
     }
 
+    @objc
+    static func migrateStoreToLatestModel(storeURL: URL, modelURL: URL) {
+        guard FileManager.default.fileExists(atPath: storeURL.path) else {
+            DDLogInfo("No store exists at URL \(storeURL).  Skipping migration.")
+            return
+        }
+
+        guard let metadata = try? NSPersistentStoreCoordinator.metadataForPersistentStore(ofType: NSSQLiteStoreType, at: storeURL, options: nil) else {
```

Comment:
not sure if we want to catch the error here too?

Reply:
I'll remove the entire method for now.